pyproject.toml

```diff
@@ -1,10 +1,26 @@
 [project]
 name = "ome-zarr-models"
 dynamic = ["version"]
-description = "Data models for OME-Zarr"
+description = "A minimal Python package for reading OME-Zarr (meta)data "
 readme = "README.md"
 requires-python = ">=3.11"
 dependencies = ["zarr<3", "pydantic", "pydantic-zarr"]
+maintainers = [
+  {name = "David Stansby"}
```

Comment:
@d-v-b, @lorenzocerrone you should definitely open a PR to add your self here if you want to be listed as a maintainer on PyPI!